codalab/worker_manager/aws_batch_worker_manager.py

```diff
@@ -32,11 +63,6 @@ def get_worker_jobs(self):
 
     def start_worker_job(self):
         image = 'codalab/worker:' + os.environ.get('CODALAB_VERSION', 'latest')
-        logger.debug('Starting worker with image {}'.format(image))
```

Comment:
Why remove the debug statement?

Reply:
It just felt strange to me that we print the image as a debug statement. Doesn't feel too informative to me. Will re-add with the ID as well.